diff -ura data/usr/lib/systemd/system/vanta.service data.new/usr/lib/systemd/system/vanta.service
--- data/usr/lib/systemd/system/vanta.service	2025-05-27 08:19:12.000000000 -0700
+++ data.new/usr/lib/systemd/system/vanta.service	2025-11-06 09:47:14.734898045 -0800
@@ -1,14 +1,12 @@
 [Unit]
-Description=Vanta monitoring software
-After=network.service syslog.service
+Description=Vanta Device Monitor
+After=network.service
 
 [Service]
-TimeoutStartSec=0
+Type=exec
 ExecStart=/var/vanta/metalauncher
 Restart=on-failure
+BindPaths=/etc/os-release.ubuntu_2510:/etc/os-release
-KillMode=control-group
-KillSignal=SIGTERM
 
 [Install]
 WantedBy=multi-user.target
-Alias=vanta.service
\ No newline at end of file
